build.sbt

```diff
@@ -397,7 +397,8 @@ def mimaSettings(moduleName: String) =
           exclude[IncompatibleMethTypeProblem]("cats.arrow.FunctionKMacros.lift"),
           exclude[MissingTypesProblem]("cats.arrow.FunctionKMacros$"),
           exclude[IncompatibleMethTypeProblem]("cats.arrow.FunctionKMacros#Lifter.this"),
-          exclude[IncompatibleResultTypeProblem]("cats.arrow.FunctionKMacros#Lifter.c")
+          exclude[IncompatibleResultTypeProblem]("cats.arrow.FunctionKMacros#Lifter.c"),
+          exclude[ReversedMissingMethodProblem]("cats.Functor.ifF")
```

Comment:
This will break binary compatibility on 2.11. you should add this method as an extension method instead (see the bincompat traits that already exist for other type classes)

Reply:
I just realized 2.11 has been dropped on master so it doesn't matter. You should be able to rebase onto master and remove this line.

Reply:
I wasn't able to rebase to recent master so I force pushed my changes.